### DISCUSSION TITLE
Include a tool for documentation

### DISCUSSION THREAD

**Discussion by fulldecent** [Kong/insomnia#6240] [Ideas]
I downloaded the app, created a request and ran it.

But I don't see anywhere to make documentation for this API. Like an HTML file with inputs/outputs defined, or some OpenAPI specification (or both!).

If this is a paid feature or needs to be set up, please say that in the app since this is probably something a lot of people want.

<img width="1837" alt="Screenshot 2023-08-08 at 12 29 52" src="https://github.com/Kong/insomnia/assets/382183/a1fe20db-ce3c-4ca1-a824-8d6f2eed1cb1">

**Comment by carmolim** [comment]
I think you create a collection, to create an OpenAPI specification   you need to create an Document there you can desing your API create requests and tests

**Comment by fulldecent** [comment]
In this example, I believe I did create a collection. 
<img width="1837" alt="259167045-a1fe20db-ce3c-4ca1-a824-8d6f2eed1cb1" src="https://github.com/Kong/insomnia/assets/382183/d9cbc2b4-76db-413a-a4d6-da008cb29fab">

But also I tried again and definitely made a collection this time.
<img width="1392" alt="Screenshot 2023-08-13 at 23 36 09" src="https://github.com/Kong/insomnia/assets/382183/e351d8ee-ba62-4e56-b4b6-56a81df1e422">

From there I didn't see anywhere to create a document.

**Comment by carmolim** [comment]
Here are the steps

<img width="1614" alt="image" src="https://github.com/Kong/insomnia/assets/2863161/e5c8f1a5-11fc-4e6d-8e9d-84b0a6b3cf4b">
<img width="290" alt="image" src="https://github.com/Kong/insomnia/assets/2863161/75159c72-0537-40dd-ba7a-0d5ecec7b390">
<img width="1700" alt="image" src="https://github.com/Kong/insomnia/assets/2863161/582c1ded-5ad7-4b4b-b9d9-789b2aff6acc">

**Comment by fulldecent** [comment]
Aha, cool, thank you!
